tests/kola/rpm-ostree/replace-rt-kernel

```diff
@@ -69,13 +65,14 @@ case "${AUTOPKGTEST_REBOOT_MARK:-}" in
         x86_64)
             tmpdir=$(mktemp -d -p /var/tmp)
             pushd "${tmpdir}"
+            #setup repos
+            runv rm -rf /etc/yum.repos.d/*
+            runv curl -v "${repo_url}" -o /etc/yum.repos.d/cs.repo
+            runv curl -L https://centos.org/keys/RPM-GPG-KEY-CentOS-Official > /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Official
+            runv sed -i 's|^gpgkey.*|gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Official|' /etc/yum.repos.d/cs.repo
             runv rpm-ostree override reset -a
-            runv curl -sSLf --remote-name-all "${baseurl}/NFV/${basearch}/os/Packages/kernel-rt"{-core,-modules,-modules-extra,-kvm}"-${target_kver_rt}.${basearch}.rpm"
-            args=()
-            for x in ./*.rpm; do
-                args+=(--install ./"${x}")
-            done
-            runv rpm-ostree override remove kernel{,-core,-modules,-modules-extra} "${args[@]}"
+            args=(--install kernel-rt)
```

Comment:
We don't actually do `kernel-rt` in OCP because it's a metapackage that drags in too much stuff, see https://github.com/openshift/machine-config-operator/blob/dadba5bda3ad2051050a93e7dd0ea9e7f1324b21/pkg/daemon/update.go#L1174

So I'd probably copy that for now.